components/StyledCarousel.js

```diff
@@ -166,7 +167,7 @@ const StyledCarousel = props => {
               {React.Children.map(children, (child, index) => {
                 return (
                   <CarouselSlot order={getOrder(index)} mx={2}>
-                    {child}
+                    {props.fitToContent ? <Span display={activeIndex !== index ? 'none' : null}>{child}</Span> : child}
```

Comment:
Unless I'm missing something, `child` can/will contain block elements. Can't use `span` as a parent for these